packages/azure/src/__snapshots__/azure-openai-provider.test.ts.snap

```diff
@@ -1,5 +1,70 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`chat > doStream > should set .modelId for model-router request 1`] = `
+[
+  {
+    "type": "stream-start",
+    "warnings": [],
+  },
+  {
+    "id": "chatcmpl-CYPS1lijGoK8gd9lYzY3r9Sx50nbt",
+    "modelId": "gpt-5-nano-2025-08-07",
+    "timestamp": 2025-11-05T04:30:21.000Z,
```

Comment:
this is the relevant part. Without the fix all 3 properties would be unset (timestamp would be `1970-01-01T00:00:00.000Z`